src/elements/Icon/Icon.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component, PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+
+export default class Icon extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+  }
+
+  static _meta = {
+    library: META.library.semanticUI,
+    name: 'Icon',
+    type: META.type.element,
+  }
+
+  render() {
+    const { className } = this.props
+    const classes = cx(
+      'sd-icon',
+      className,
+      'icon',
+    )
+    return (
+      <i {...this.props} className={classes} />
```

Comment:
One thing I've seen with a lot of `<Icon />` components is the usage of props for icon type rather than classes, that way it's not just a no-value-added wrapper over an `<i>` element. It also makes it quicker (read: less code) to create them (compare `<Icon heart />` vs. `<Icon className='heart' />`)

Food for thought.